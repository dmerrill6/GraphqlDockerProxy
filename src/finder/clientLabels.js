export const TOKEN = 'gqlProxy.token'
export const URL = 'gqlProxy.url'
export const NAMESPACE = 'gqlProxy.namespace'
