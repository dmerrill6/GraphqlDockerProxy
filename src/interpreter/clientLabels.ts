export const TOKEN:string = 'gqlProxy.token'
export const URL:string = 'gqlProxy.url'
export const NAMESPACE:string = 'gqlProxy.namespace'
