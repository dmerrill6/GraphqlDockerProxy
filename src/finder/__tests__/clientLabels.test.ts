import * as clientLabels from '../clientLabels';
describe('the labels to known thats to include', () => {
  it('Snapshot the labels', () => {
    expect(clientLabels).toMatchSnapshot();
  });
});
