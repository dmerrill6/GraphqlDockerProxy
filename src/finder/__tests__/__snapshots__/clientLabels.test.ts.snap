// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the labels to known thats to include Snapshot the labels 1`] = `
Object {
  "NAMESPACE": "gqlProxy.namespace",
  "TOKEN": "gqlProxy.token",
  "URL": "gqlProxy.url",
}
`;
